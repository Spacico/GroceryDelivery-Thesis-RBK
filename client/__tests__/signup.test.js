// import React from 'react';
// import Signup from '../components/signup';
// // import TestUtils from 'react-dom/test-utils';;
// import renderer from 'react-test-renderer';
// import expect from 'expect';
// // import { configure } from 'enzyme';
// // import Adapter from 'enzyme-adapter-react-15';
// // const Enzyme = require('enzyme');
// // import { shallow, mount, render } from 'enzyme';
// // Enzyme.configure({ adapter: new Adapter() });

// describe('App ReactNativeTesting', () => {

// it('renders without crashing', () => {
//   const rendered = renderer.create(<Signup />).toJSON();
//   expect(rendered).toBeTruthy();
// });

// it('general test ......', () => {
//  const TextInput = require('TextInput');
//   const tree = renderer
//     .create(<TextInput autoCorrect={false} value="apple banana kiwi" />)
//     .toJSON();
//   expect(tree).toMatchSnapshot();
// })



// })
