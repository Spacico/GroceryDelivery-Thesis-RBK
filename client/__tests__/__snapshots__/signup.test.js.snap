// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App ReactNativeTesting general test ...... 1`] = `
<TextInput
  autoCorrect={false}
  value="apple banana kiwi"
/>
`;
